src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -551,7 +549,7 @@ export default class PanoImageRenderer extends Component {
 
 		mat4.identity(this.mvMatrix);
 		mat4.rotateX(this.mvMatrix, this.mvMatrix, -glMatrix.toRadian(pitch));
-		mat4.rotateY(this.mvMatrix, this.mvMatrix, -glMatrix.toRadian(yaw - this._initialYaw));
+		mat4.rotateY(this.mvMatrix, this.mvMatrix, -glMatrix.toRadian(yaw));
```

Comment:
why remove this._initialYaw

Reply:
Because it is substitute by changing initial shader data for the performance and code simplicity.

Ref `src/PanoImageRenderer/renderer/SphereRenderer.js`